### ISSUE TITLE
Incorrect logic in get_documens step

### ISSUE THREAD

**Issue by mrozpawel** [echr-od/ECHR-OD_process#170]
Logic of this code is not compatible to it's description
```
 Get documents according to the specified list
        :param id_list: list of document id
        :type id_list: [str]
        :param folder: path where to save the documents
        :type folder: str
    --->:param update: overwrite existing documents<---
        :type: bool
``` 
```
def get_documents_step(doc_id, progress, task):
        if doc_id[1]:
            filename = "%s.docx" % (doc_id[0].strip())
        else:
            filename = "%s.pdf" % (doc_id[0].strip())
        filename = os.path.join(folder, filename)
    --->if not update or not os.path.isfile(filename): <---
```

**Comment by mrozpawel** [comment]
Also, another task is to handle cases if `id_list` in empty

**Comment by mrozpawel** [comment]
Fixed with #171
